tideways/by-site.php

```diff
@@ -0,0 +1,18 @@
+#!/bin/env php    
+<?php 
+if(is_file('/srv/tideways.json')) {
+    unlink('/srv/tideways.json');
+}
+
+$vvv_config = $argv[1];
+$config     = yaml_parse_file($vvv_config);
```

Comment:
oo I didn't know PHP had this natively, we should add a check if this file actually exists though